spec/models/form_profile_spec.rb

```diff
@@ -698,10 +698,12 @@
         'addressLine1' => street_check[:street],
         'addressLine2' => street_check[:street2],
         'city' => user.va_profile[:address][:city],
-        'stateCode' => user.va_profile[:address][:state],
-        'countryName' => 'USA',
-        'zipCode' => user.va_profile[:address][:postal_code][0..4]
-      }
+        'state' => user.va_profile[:address][:state],
```

Comment:
I see that you adjusted the stateCode, CountryName and zipCode mappings in [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/5041/files#diff-2289ed0baec2d282157fc61580876bb7R701-R703) and lines 701-703 revert those changes.

Reply:
@LindseySaari So I changed that because I am getting unit test failures with it the way it was on the build for this PR. I am not really sure what is causing the unit tests to fail.

Reply:
It looks like you have a few typos in the spec. I commented inline